rtengine/profilestore.cc

```diff
@@ -261,10 +261,10 @@ bool ProfileStore::parseDir (Glib::ustring& realPath, Glib::ustring& virtualPath
     return fileFound;
 }
 
-int ProfileStore::findFolderId (const Glib::ustring &path)
+int ProfileStore::findFolderId (const Glib::ustring &path) const
 {
     // initialization must have been done when calling this
-    for (std::vector<Glib::ustring>::iterator i = folders.begin(); i != folders.end(); ++i) {
+    for (std::vector<Glib::ustring>::const_iterator i = folders.begin(); i != folders.end(); ++i) {
```

Comment:
One could use a simple index here, as it is returned later.

Reply:
Fixed.